### ISSUE TITLE
[Feature Request] Make configurable the min size (width and height) for "float_small_windows"

### ISSUE THREAD

**Issue by protiumx** [ianyh/Amethyst#1565]
**Is your feature request related to a problem? Please describe.**
Many apps show small windows that most of the time are not square or are a bit bigger than this threshold.

**Describe the solution you'd like**
I'd like to be able to configure the width and height to which a windows is consider "small".

**Describe alternatives you've considered**
I haven't been able to find a workaround. Usually this popup windows are not resizable.

**Additional context**
I'm using a 1440p ultra wide screen

**Comment by DanCates** [comment]
Agreed. This would be fantastic to have more control over this feature. I struggle with Get Info windows all the time in Finder as well as many other "small" windows that are not small enough for the inherent criteria.

**Comment by shaleh** [comment]
https://github.com/ianyh/Amethyst/blob/1defe1a997b38356cdef694f19f10d78cf625e62/Amethyst/Model/Window.swift#L264

Does not look too crazy. The current logic is anything smaller than 500x500.
